.github/workflows/ci-windows.yml

```diff
@@ -102,3 +100,6 @@ jobs:
     - name: Build libavif (ninja)
       working-directory: ./build
       run: ninja
+    - name: Run AVIF Tests
+      working-directory: ./build
+      run: ctest -j 2
```

Comment:
Please let me know if you prefer a larger value for the `-j` option.

Reply:
This is ok, though it's unfortunate we can't pull that from the environment. It would need to be different for all 3 os images in that case.